controllers/azurejson_machinetemplate_controller.go

```diff
@@ -48,6 +48,7 @@ type AzureJSONTemplateReconciler struct {
 	Log              logr.Logger
 	Recorder         record.EventRecorder
 	ReconcileTimeout time.Duration
+	WatchFilterValue string
```

Comment:
Needs `WithEventFilter(predicates.ResourceHasFilterLabel(r.Log, r.WatchFilterValue)).` in line 58